Makefile.shared

```diff
@@ -154,10 +154,6 @@ else
 endif
 
 installsrc:
-ifndef XCODE_TARGET
-	@$(call invoke_xcode,,,-alltargets installsrc SRCROOT="$(SRCROOT)$(PATH_FROM_ROOT)")
-else
-	@$(call invoke_xcode,,,installsrc SRCROOT="$(SRCROOT)$(PATH_FROM_ROOT)")
-endif
+	xcodebuild$(if $(XCODE_TARGET), -alltargets) installsrc SRCROOT="$(SRCROOT)$(PATH_FROM_ROOT)"
```

Comment:
self-nit: I am not sure I understand what `-alltargets` does, since I belive installsrc operates on the entire project. Maybe we don't need it?

Reply:
Not sure. I cc'ed you on an older bug where something related was changed, but I'm not sure if it's actually relevant.

Reply:
Makes sense. I think it explains why we got this flag: The common Xcode settings pass a `-target` selection when building libwebrtc.xcodeproj for watchOS. Now, since we won't be passing the common options to installsrc, we can omit this.